openerp/addons/base/res/res_company_view.xml

```diff
@@ -50,7 +50,7 @@
                         </page>
                         <page name="configuration" string="Configuration">
                             <group>
-                                <group name="account_grp" string="Accounting"/>
```

Comment:
I just took the 2 branches and it still visible in :

/account_sepa/views/res_company_view.xml
/l10n_us_check_printing/views/res_company_view.xml

I'll check it.